libcxx/utils/ci/vendor/android/Dockerfile.emulator

```diff
@@ -16,7 +16,7 @@ RUN apt-get update && apt-get install -y \
     unzip \
     && rm -rf /var/lib/apt/lists/*
 
-ENV ANDROID_HOME /opt/android/sdk
+ENV ANDROID_HOME=/opt/android/sdk
```

Comment:
We don't test the Dockerfile changes in our CI. Did you verify this works correctly?

Reply:
Yes, I tested it by running `run-buildbot android-ndk-33-goog-x86_64` inside `vendor/android/run-buildbot-container`. To do that, I first needed to apply #99623 and comment out the broken LLVM-16-on-noble step, as explained on that PR.